lib/candlex/native.ex

```diff
@@ -4,7 +4,9 @@ defmodule Candlex.Native do
   mix_config = Mix.Project.config()
   version = mix_config[:version]
   source_url = mix_config[:package][:links]["GitHub"]
-  mode = if Mix.env() in [:dev, :test], do: :debug, else: :release
+  # Test with release mode temporarily
+  # mode = if Mix.env() in [:dev, :test], do: :debug, else: :release
+  mode = :release
```

Comment:
Apparently the failure `attempt to add with overflow` in first commit is gone if `cargo` run on release mode instead of debug.

Need to dug deeper to understand what's causing the error in debug mode.

Reply:
Seems there's a u32 addition of `3131444833 + 4143857739` causing the panic because of the `attempt to add with overflow`.
Coming from `Nx.Random.normal` call in first call to init random params.

Reply:
Narrowed it down to a problem when calculating the next random key.
Planning to open separate PR with test and fixes for that.

Reply:
There's an actual bug in `Nx.Random`.
https://github.com/mimiquate/candlex/pull/32.

Reply:
#32 merged into `main`.

Now updated this PR with `main`.